### PR TITLE
OCPBUGS-16554: update dependencies to get rid of goproxy

### PR DESCRIPTION
**Pull request by mfojtik**

### CODE REVIEW

File: pkg/operator/starter.go [hunk 1]
```diff
@@ -180,7 +180,9 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 		controllerConfig.EventRecorder,
 	).WithAPIServiceController(
 		"openshift-apiserver",
-		func() ([]*apiregistrationv1.APIService, error) { return apiServices(), nil },
+		func() (enabled []*apiregistrationv1.APIService, disabled []*apiregistrationv1.APIService, err error) {
```
[the remark this answers is not in the document]
Comment (ingvagabund) [reply pkg/operator/starter.go:183]: Looks good otherwise

[the remark this answers is not in the document]
Comment (mfojtik) [reply pkg/operator/starter.go:183]: @ingvagabund very fast

[the remark this answers is not in the document]
Comment (ingvagabund) [reply pkg/operator/starter.go:183]: I'll rebase then. feel free to go ahead.

**Pull request merged**